File: strings/notes.py
# Julia Properzi, strings notes python

#string is a data type that holds any info surrounded by quotation marks "" ''

note = "Vienna's class"

#name = (input("What is your first name?\n").strip().capitalize() )

#print(f"Hello {name} wlcome to my program")
#print("This is your name" + name)

sentence = "The quick brown fox jumps over the lazy dog"

#print(len(sentence))
#print(sentence[4])
start = sentence.find("brown")
length = len("brown fox")
#print(sentence.find("brown"))
#print(sentence[10:19])
print(sentence[start:start+length])

File: strings/notes.c
// Julia Properzi, Strings Notes C
#include <stdio.h>
#include <string.h>

char subject[50];
char name[] = "Victoria";
char sentence[] = "The quick brown fox jumps over the lazy dog.";

int main(void){
   // printf("What class are you in?\n");
    //scanf("%s", subject);
    //fgets(subject, sizeof(subject), stdin);
    //printf("You are in %s, that is a cool class", subject);
    //name[0] = 'T';
    //name[1] = 'o';
    //name[2] = 'r';
    //name[3] = 'i';
    //printf("%s", name);
    //printf("%lu\n", sizeof(sentence)); 
    //printf("%d\n", strlen(sentence));
    char one[] = "Hello ";
    char two[] = "World!";
    char three[] = "Welcome to my program. ";
    printf("%s\n", one);
    strcat(one, two);
    printf("%s\n", one);
    strcat(three, one);
    printf("%s", three);
    return 0;
}